### PR TITLE
Fixed UAudioManager Editor Bug and added Test Scene for UAudioManager

### LINKED ISSUE #32: Create test scene for SpatialSound\UAudioManager
based on Holograms-220 from the HolographicAcademy

### PR DESCRIPTION
**Pull request by janholo**
Fixes #32 

during renaming of the UAudioManager Module (commit: a473ee2) the Names in the Editor Scripts were not changed acordingly. This caused errors in the editor.

I changed the variable names and also added a test scene for the UAudioManager.

### CODE REVIEW

File: Assets/HoloToolkit-Tests/SpatialSound/Scripts/UAudioManagerTest.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿using System;
```
Comment (Zod-) [inline Assets/HoloToolkit-Tests/SpatialSound/Scripts/UAudioManagerTest.cs]: Add License Header

File: Assets/HoloToolkit-Tests/SpatialSound/Scripts/UAudioManagerTest.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+using System.Collections;
+using HoloToolkit.Unity;
+using UnityEngine;
+
+public class UAudioManagerTest : MonoBehaviour
+{
+	void Start ()
```
Comment (Zod-) [inline Assets/HoloToolkit-Tests/SpatialSound/Scripts/UAudioManagerTest.cs]: Use private access modifier

**Comment by janholo** [comment]
Done, 
in file AudioClip.cs, AudioContainer.cs and maybe a few others is the naming still incosistent. (public variable starts with lowercase). Should this also be changed or do you wanna keep it that way?

**Comment by janholo** [comment]
IMHO: this just bloats the code unnecessary ;-)

**Comment by StephenHodgson** [comment]
>In file AudioClip.cs, AudioContainer.cs and maybe a few others is the naming still incosistent. (public variable starts with lowercase). Should this also be changed or do you wanna keep it that way?

No, they should also match the PascalCase naming conventions of the project.

**Comment by Zod-** [comment]
Ok, it doesn't actually update the serialization name to the new. It will just keep the old name. I'm not so sure about this then.

**Comment by janholo** [comment]
Ok, 
I hope Resharper got everything.
There were more changes needed than I thought

**Comment by Zod-** [comment]
Does this solve #32? That issue has been around for a long time.

**Comment by StephenHodgson** [comment]
Indeed

**Pull request merged**